Carousal

FatumAbdullahi opened the pull request:
close #22

ahmedhamodi commented on src/Components/LandingPage/Carousal.js:
```diff
@@ -0,0 +1,211 @@
+      text-align:center;
+     }
+    
+   `} src="https://upload.wikimedia.org/wikipedia/commons/f/f6/Sean_Cunningham_Head_Shot.jpg"/>
```
Can you try to save images to our `src/Components/Images` folder and reference them from there instead of external sites? Probably make a subfolder in Images for user testimonials.

FatumAbdullahi replied:
Should be done!

ahmedhamodi commented on src/Components/LandingPage/Carousal.js:
```diff
@@ -0,0 +1,211 @@
+      }
+    
+    
+   `} src="https://images.squarespace-cdn.com/content/v1/5c64375df4e5314e7985012c/1565025240158-JV7FOIODKHPLAZOIRGRN/ke17ZwdGBToddI8pDm48kC-KQ8StUrUTl7SMDbWuteJZw-zPPgdn4jUwVcJE1ZvWQUxwkmyExglNqGp0IvTJZamWLI2zvYWH8K3-s_4yszcp2ryTI0HqTOaaUohrI8PI2jhaLsRRsAwInQ5q0zH86wR4v5ScThrPpqJ8W9VBf8cKMshLAGzx4R3EDFOm1kBS/marcus+square+head+shot.jpg?format=750w"/>
```
Same as above

FatumAbdullahi replied:
Done!

ahmedhamodi commented on src/Components/LandingPage/Quote.js:
```diff
@@ -0,0 +1,268 @@
+import React, { useState } from 'react';
+import {Component} from "react";
+import { css, cx } from 'emotion'
```
I'm not able to run this code locally because of this import - where are you getting this? I'm assuming it was something you npm installed? If so, you need to push that modified package.json file so we can install it on our own machines.

ahmedhamodi commented on src/Components/LandingPage/Carousal.js:
```diff
@@ -0,0 +1,211 @@
+import React, { useState } from 'react';
+import {Component} from "react";
+import { css, cx } from 'emotion'
```
I'm not able to run this code locally because of this import - where are you getting this?

ahmedhamodi commented on src/Components/LandingPage/Carousal.js:
```diff
@@ -0,0 +1,211 @@
+  HandleRightArrowClick=(event)=>{
+    if (this.state.active<3){
+      this.setState({current:quotes[parseInt(this.state.active)+1],active:parseInt(this.state.active)+1})
+    }  
```
Can you make this constantly looping? ie. if active is at 3, reset to image at 0.

FatumAbdullahi replied:
Done!

ahmedhamodi commented on src/Components/LandingPage/Carousal.js:
```diff
@@ -0,0 +1,211 @@
+  HandleLeftArrowClick=(event)=>{
+    if (this.state.active>0){
+      this.setState({current:quotes[this.state.active-1],active:this.state.active-1})
+    } 
```
Like wise here, if going left at 0, show the image at the end.

FatumAbdullahi replied:
Done!

ahmedhamodi commented on src/Components/LandingPage/Carousal.js:
```diff
@@ -0,0 +1,211 @@
+
+  HandleSetClick=(event)=>{
+    this.setState({active:event.target.getAttribute("data-image"),current:quotes[event.target.getAttribute("data-image")]})
+    console.log(this.state.current)
```
Do you want to keep this for debugging or can it be removed?

FatumAbdullahi replied:
Removed, forgot

ahmedhamodi commented on src/Components/LandingPage/Quote.js:
```diff
@@ -0,0 +1,268 @@
+      display: 'None'
+  }
+}
+const quotes={
```
You have these quotes twice in code. Is this needed? If so, move it to a different file and include it instead of having it present in both. The reason being, when we change them in the future we only want to change in one place.

ahmedhamodi commented on src/Components/LandingPage/Quote.js:
```diff
@@ -0,0 +1,268 @@
+
+  
+  }
+  //setCurrent(value){this.setState({current:value})}
```
Remove commented out, unneeded code.

ahmedhamodi commented on src/Components/LandingPage/Quote.js:
```diff
@@ -0,0 +1,268 @@
+    //this.setCurrent(quotes[event.target.getAttribute("data-quote")]) ;
+    this.setState({active:event.target.getAttribute("data-quote"),current:quotes[event.target.getAttribute("data-quote")]})
+  // this.setActive(event.target.getAttribute("data-quote"));
+    console.log(event.target.getAttribute("data-quote"));
```
These logs are likely not needed.

ahmedhamodi commented on src/Components/LandingPage/Quote.js:
```diff
@@ -0,0 +1,268 @@
+      `}
+      
+      
+      style={{paddingTop: '30px'}}><b>What our members have to say</b></h1>
```
All these appears duplicated, and I cannot run it locally to even test it. Can you go through this code yourself and remove unneeded code and clean it up? The code spacing and tabbing is very hard to read.

FatumAbdullahi commented:
As suggested by:
moved the quotes to a separate file
properly tab nested components
used local images 
looped carousal
changed the colour of highlight to gold

ahmedhamodi commented:
Reopening
